content/post/manage-redis-secret-engine-using-kubevault-in-aws/index.md

```diff
@@ -0,0 +1,587 @@
+---
+title: Leveraging KubeVault to Manage the Redis Secret Engine in Amazon EKS - A Step-by-Step Guide
+date: "2023-01-27"
+weight: 14
+authors:
+- Dipta Roy
+tags:
+- amazon
+- aws
+- cloud-native
+- database
+- dbaas
+- eks
+- kubedb
+- kubernetes
+- kubevault
+- redis
+- s3
+- secrets
+---
+
+## Overview
+
+KubeVault provides various ways to configure your Vault deployment. You can pick your preferred Storage Backend, Unsealer Mode, TLS Mode, Secret Engines that you want to allow to attach with this VaultServer, Termination Policy to prevent accidental deletion or clean-up Vault deployment in a systematic way, Monitoring, etc. KubeVault is a Git-Ops ready, production-grade solution for deploying and configuring Hashicorp's Vault on Kubernetes. You can find the guides and detailed information in [KubeVault](https://kubevault.com/).
```

Comment:
`KubeVault is a Git-Ops ready, production-grade solution for deploying and configuring Hashicorp’s Vault on Kubernetes. `

Can we move this line as the first line ?